pyproject.toml

```
@@ -162,5 +165,5 @@ require_poetry = true
 """
 
 [build-system]
-requires = ["poetry-core>=1.0.0,<2"]
+requires = ["poetry-core>=2.0.0"]
```

```suggestion
requires = ["poetry-core>=2.0.0,<3"]
```
Safer to have the upper bound to avoid surprises.